src/scripts/validation.js

```diff
@@ -57,7 +59,7 @@ const disableButton = (buttonElement, config) => {
   buttonElement.classList.add(config.inactiveButtonClass);
```

Comment:
* You need to **`disable`** the submit button in the `card form` if the inputs are **empty**. It’s needed because I can click the button and add an empty card, but it should **not** happen. After the first submission, the button gets activated, and you need to **deactivate** it again. Even if I can't add an empty card, the button should look **disabled** (gray) anyway and have a `disabled` attribute.


Please, **export** the `disableButton` functon from `validation.js`, to be able to use it in `index.js`

```js

export const disableButton = (buttonElement, config) => {

```